TradeItIosTicketSDK2/Constants.swift

```diff
@@ -0,0 +1,27 @@
+@objc class TradeItNotificationConstants: NSObject {
+    // TODO: Move existing notification center constants here
+
+    // MARK: Notification names
+    public static let nameViewDidAppear = TradeItNotification.Name.viewDidAppear
+    public static let nameErrorShown = TradeItNotification.Name.errorShown
+
+    // MARK: Notification UserInfo keys
+    public static let userInfoKeyView = TradeItNotification.UserInfoKey.view
+    public static let userInfoKeyErrorTitle = TradeItNotification.UserInfoKey.errorTitle
+    public static let userInfoKeyErrorMessage = TradeItNotification.UserInfoKey.errorMessage
+}
+
+public struct TradeItNotification {
```

Comment:
I don't think Swift structs are available in obj-c

Reply:
See the @objc class above.